src/internal/ReplaySubject.ts

```diff
@@ -35,19 +41,18 @@ export class ReplaySubject<T> extends Subject<T> {
   }
 
   private nextInfiniteTimeWindow(value: T): void {
-    const _events = this._events;
```

Comment:
This was done to prevent additional property lookups and the change should be reverted.